src/main/java/org/opensearch/knn/index/IndexUtil.java

```diff
@@ -286,4 +303,13 @@ public static boolean isSharedIndexStateRequired(KNNEngine knnEngine, String mod
         return JNIService.isSharedIndexStateRequired(indexAddr, knnEngine);
     }
 
+    public static String getIndexDescription(FieldInfo fieldInfo) throws IOException {
```

Comment:
nit: You are extracting information from FieldInformation here. this in indexUtil seems out of place. Maybe consider creating a class `FieldInfoExtractor` and having a method there? That class will have a defined purpose. Having a generic Util class has overloaded responsibilities, we should start moving away from it